streams/runners/storm/common/src/main/java/com/hortonworks/streamline/streams/storm/common/StormJaasCreator.java

```diff
@@ -0,0 +1,74 @@
+package com.hortonworks.streamline.streams.storm.common;
+
+import org.apache.commons.io.FileUtils;
+
+import javax.security.auth.login.AppConfigurationEntry;
+import javax.security.auth.login.Configuration;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class StormJaasCreator {
+    public static final String STORM_JAAS_CONFIG_TEMPLATE = "storm_jaas.config_template";
```

Comment:
That's template for JAAS config file, and it replaces keytab, principal with streamline keytab and principal, and serviceName with service name of Nimbus of Storm cluster.